adapters/tuzlanski.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+import { REQUEST_TIMEOUT } from '../lib/constants';
+import { default as baseRequest } from 'request';
+const request = baseRequest.defaults({timeout: REQUEST_TIMEOUT});
+import cheerio from 'cheerio';
+import Coordinates from 'coordinate-parser';
```

Comment:
I'm only using `coordinate-parser` to convert this format: `44°31´55´´N` to decimal degrees. I don't see how Turf does that.